### ISSUE TITLE
Write a test in example_component_test.hpp testing the sendMsg method 

### ISSUE THREAD

**Issue by LuxMiranda** [SmallSatGasTeam/OpenSPA#29]
- Work out of the `tutorial` branch
- Initialize a component in the test and call the sendMsg method
- Understand and utilize the file's test fixture
- The test should pass if `bool sent` is true

**Comment by CommanderScotty** [comment]
Hey I'm having a bit of trouble figuring out how to initialize the component.  I can't seem to find a constructor to call for one.  Could someone tell me what file its in / how to use it?

**Comment by LuxMiranda** [comment]
Yeah man! @CommanderScotty It's [this file right here.](https://github.com/SmallSatGasTeam/OpenSPA/blob/tutorial/lib/example_component.hpp) In that file there's a bunch of commented code that shows you exactly how to initialize the example component. You can copy/paste it right into the test fixture's SetUp function for the most part :)
